src/cmd/root.go

```diff
@@ -68,6 +72,18 @@ func (o *outputFormat) Type() string {
 var rootCmd = NewZarfCommand()
 
 func preRun(cmd *cobra.Command, _ []string) error {
+	// Configure user defined Features
+	err := setupFeatures(features)
```

Comment:
I am looking back to the proposal to understand how this is intended to be used throughout the CLI. 

Should this operation be a `PersistentPreRun` instead - allowing for `setupFeatures` to be defined in a single location `root.go`?

Reply:
This is a good callout - in our case the function named "preRun" is being executed as a PersistentPreRun. The naming is a bit off but the functionality should be there - otherwise other setup like logger wouldn't be applying on subcommands.

Reply:
ah that is deceiving but I see it now! looks good.